frontend/public/components/app.tsx

```diff
@@ -321,10 +325,10 @@ const AppRouter = () => {
   // Treat the authentication error page as a standalone route. There is no need to render the rest
   // of the app if we know authentication has failed.
   return (
-    <Router history={history} basename={window.SERVER_FLAGS.basePath}>
```

Comment:
turns out the reason we had to monkeypatch the `history` object was because `basename` wasn't even a valid prop for the router component. :facepalm:

Reply:
It looks like it might be available on `BrowserRouter`?

We have had requests to allow setting the base path of the console app, although it's not supported today through any configuration. I'm debating if we try to keep this compatibility. If we know it doesn't work, it would be good to remove the CLI flag from the backend.